libs/counterfactuals/.eslintrc.json

```diff
@@ -13,6 +13,21 @@
     {
       "files": ["*.js", "*.jsx"],
       "rules": {}
+    },
+    {
+      "files": ["**/getBubbleChartOptions.ts"],
+      "rules": {
+        "dot-notation": "off"
+      }
+    },
+    {
+      "files": [
+        "**/CounterfactualComponent.tsx",
+        "**/LargeCounterfactualChart.tsx"
+      ],
+      "rules": {
+        "max-lines": "off"
```

Comment:
Any particular reason for turning this off?

Reply:
As most of the functions has state changes, I and Tong tried to reduce as much as possible. But it was really difficult to reduce lines to <300. Hence added this rule. We will discuss about this with Ke in meeting.